src/ini.pest

```diff
@@ -1,13 +1,17 @@
-WHITESPACE = _{ " " }
+ini = { SOI ~ (section | pair | comment | blank_line)* ~ ws? ~ EOI }
```

Comment:
- Only the prelude (root level) can have sections in it.
- The `ws?` at the end accounts for a final line in which it only has spaces, but no final newline.

Reply:
Seems to be true.